website/sales/admin/shift_admin.py

```diff
@@ -108,21 +108,26 @@ class ShiftAdmin(admin.ModelAdmin):
         "end",
         "active",
         "product_list",
+        "managers",
         "product_sales",
         "num_orders",
         "total_revenue",
         "locked",
-        "managers",
     )
 
     readonly_fields = (
-        "title",
```

Comment:
Title field should thus never be read only?

Reply:
nope (unless the shift is locked, but then the whole change permission is disabled)